networks.py

```diff
@@ -124,26 +125,30 @@ def get_nll(self, outputs, target, sigma=1.):
         if self.mode == 'regression':
             ll = torch.distributions.Normal(outputs, sigma).log_prob(target).sum()
         elif self.mode == 'classification':
-            ll = nn.CrossEntropy()
+            ll = -nn.CrossEntropyLoss()(outputs, target)
```

Comment:
are we sure this should be negative? on line 131, the code is returning -ll

Reply:
Yes because CrossEntropyLoss has a negative built in.

Reply:
Agreed.